src/Security/Security/help/Get-AzRegulatoryComplainceAssessment.md

```diff
@@ -1,11 +1,11 @@
 ---
 external help file: Microsoft.Azure.PowerShell.Cmdlets.Security.dll-Help.xml
 Module Name: Az.Security
-online version: https://docs.microsoft.com/powershell/module/az.security/Get-AzRegulatoryComplainceAssessment
+online version: https://docs.microsoft.com/powershell/module/az.security/Get-AzRegulatoryComplianceAssessment
```

Comment:
Description : Online version in the header of the file is incorrect. The corresponding URL should be the following: 
                https://docs.microsoft.com/powershell/module/az.security/get-azregulatorycomplainceassessment